Services/AuthenticationService/Controllers/AuthenticationController.cs

```diff
@@ -6,13 +6,21 @@
 using DTO;
 using DTO.RestRequests;
 using AuthenticationService.Interfaces;
+using Microsoft.Extensions.Logging;
 
 namespace AuthenticationService.Controllers

```

Comment:
You add this stuff only in Auth controller , who suppose to add this stuff in user service?

Reply:
This is just an example that Spartak wrote. My task was only to create a LoggerEngine and log entry logic to the database. We will need to enable logging on our own